### DISCUSSION TITLE
CCF plot for Lab 3

### DISCUSSION THREAD

**Discussion by maria-kuruvilla** [atsa-es/fish550-2023#32] [Q&A] [answered]
@mdscheuerell, when you plot the ccf between different states in a DFA model, what are you looking for? Should we be looking for patterns in the ccf model that suggests seasonality, or should we be looking at whether there is a significant correlation at certain lags? What if we see both patterns of seasonality and significant correlation?

**Comment by mdscheuerell** [comment, marked as answer]
Most of the time I wouldn't bother checking the cross correlation between 2 states from a DFA model. The example I showed [in the lab notes from Apr 20](https://atsa-es.github.io/atsa/Labs/Week%204/intro-to-dfa.html#Estimated_states_and_loadings) was just a way of exploring whether the 2 obvious seasonal cycles were in fact out of phase with one another. You wouldn't have to include a similar analysis in your write-up.
